README.md

```diff
@@ -25,54 +25,44 @@
     $ poetry install
     ```
 
-5. 🔑 Set environment variables
-
-    ```bash
-    OPENAI_API_KEY="sk-xxx"
-    ASSISTANT_ID="asst_xxx"
-
-    ASSISTANT_TITLE="Assistants API UI"
-    ENABLED_FILE_UPLOAD_MESSAGE="Upload a file" # Leave empty to disable
+5. ⚙️ Set config file `config.yaml`
+
+    ```yaml
+   app:
+      instructions: ""
+      enabled-file-upload-message: "Upload a file" # Leave empty to disable
+      openai:
+         api-key: "sk-xxx"
+         assistants: [
+            {id: "asst_xxx", title: "Assistant XXX UI"},
+            {id: "asst_yyy", title: "Assistant YYY UI"},
+         ]
```

Comment:
use `config.yaml` instead of `.env` because it's nicer to have these organised in a YAML file. However, I do understand that Streamlit cloud deployment may be affected, so it's really up to you. Can always adjust if needed. Cheers!